docs/how-to-setup-freecodecamp-locally.md

```diff
@@ -388,7 +388,7 @@ Follow these steps:
 
     This does not take any additional time than a unconventional message like 'update file' or 'add index.md'
 
-    You can learn more at about [why your should these here](https://www.conventionalcommits.org/en/v1.0.0-beta.2/#why-use-conventional-commits).
+    You can learn more at about why you should use conventional commits [here](https://www.conventionalcommits.org/en/v1.0.0-beta.2/#why-use-conventional-commits).
```

Comment:
The grammar is still off — an extraneus "at".

Reply:
thanks, missed that. updated